fmriprep/workflows/bold/base.py

```diff
@@ -352,17 +352,23 @@ def init_func_preproc_wf(bold_file, ignore, freesurfer,
 
     # if doing T2*-driven coregistration, create T2* map
     if t2s_coreg:
-        # use a joinNode to gather all preprocessed echos
-        join_split_echos = pe.JoinNode(niu.IdentityInterface(fields=['echo_files']),
-                                       joinsource='inputnode',
-                                       joinfield='echo_files',
-                                       name='join_split_echos')
-
-        # create a T2* map
-        bold_t2s_wf = init_bold_t2s_wf(echo_times=tes,
-                                       name='bold_t2s_wf',
-                                       mem_gb=mem_gb['filesize'],
-                                       omp_nthreads=omp_nthreads)
+        if not multiecho:
```

Comment:
Indentation can be reduced by one level:
```Python
if t2s_coreg and not multiecho:
    LOGGER.warning(...)
    t2s_coreg = False

if t2s_coreg:
    join_split_echos = ...
```
Please also note the `LOGGER.warning` instead of `LOGGER.warn` that is deprecated.